hibike/README.md

```diff
@@ -196,6 +196,21 @@ Device Type Enumeration:
 |         |                | 7            | enable3    | bool       | yes   | yes    |
 |  0x08   | LinearActuator |              |            |            |       |        |
 |  0x09   | ColorSensor    |              |            |            |       |        |
+|  0x0A   | YogiBear       | 0            | enable              | bool       | yes   | yes    |
```

Comment:
Have we decided to use id `0x0A` for Yogi Bear? I believe an earlier version of the hibike spec used `0x06`, since the grizzly was never a hibike device and we could kick it out of that spot.